Origin: commit: 63c6c78dc286a9379a81da0a75aa9cf7b5424d1d
Author: Eli Schwartz <eschwartz93@gmail.com>
Last-Update: 2022-06-21
Description: [PATCH] fix broken C std for gvc subproject (#146)

Historically, Meson didn't know how to set c_std for a subproject at
all. Meson 0.63.0 adds support for this, so the default_options kwarg to
subproject now begins to respect this.

budgie-desktop uses c11, but artificially sets c89 for the gvc
submodule. However, this does not actually work... the build then errors
out when trying to build
subprojects/gvc/test-audio-device-selection.p/test-audio-device-selection.c.o

Remove the faulty argument.

Fixes regression in commit 3d6d71c167dd9742cd072e7e6a3453c23ebb2658.
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

--- a/meson.build
+++ b/meson.build
@@ -148,7 +148,6 @@
 # Get gvc built before we do anything
 gvc = subproject('gvc',
     default_options: [
-        'c_std=c89',
         'static=false',
         'pkglibdir=@0@'.format(rpath_libdir),
     ],
